js/models/entity.js

```diff
@@ -0,0 +1,10 @@
+class Entity{
+    constructor(imagePath, rotation, direction,width,height,mirrorFlippedHorizontal=0){
+        this.imagePath = imagePath;
+        this.rotation = rotation;
+        this.direction = direction;
+        this.width = width;
+        this.height = height;
+        this.mirrorFlippedHorizontal = mirrorFlippedHorizontal;
```

Comment:
this property name is a bit long, but i'm not sure if there is a better name for this